prover/prover_fri_types/src/lib.rs

```diff
@@ -50,11 +48,10 @@ impl StoredObject for CircuitWrapper {
     serialize_using_bincode!();
 }
 
-#[derive(serde::Serialize, serde::Deserialize)]
+#[derive(Clone, serde::Serialize, serde::Deserialize)]
 pub enum FriProofWrapper {
     Base(ZkSyncBaseLayerProof),
     Recursive(ZkSyncRecursionLayerProof),
-    Eip4844(ZkSyncBaseProof),
```

Comment:
Just to check: Is backward compatibility of serialization a concern? Removing a variant is obviously not backward-compatible.